src/modules/automod/message.js

```diff
@@ -10,35 +10,20 @@ import config from '../../config';
 /**
  * Checks if a message contains a swear
  * @param message The message to check
- * @return {RegExpMatchArray} any match with the blacklist
+ * @return {string} any match with the blacklist
  */
 function containsSwear(message) {
-    const content = message.content
-        .toLowerCase()
-        .replaceAll(/[0ö]/g, 'o')
-        .replaceAll(/[1!|]/g, 'i')
-        .replaceAll(/3/g, 'e')
-        .replaceAll(/[4@ä]/g, 'a')
-        .replaceAll(/[5$]/g, 's')
-        .replaceAll(/[7+]/g, 't')
-        .replaceAll(/8/g, 'b')
-        .replaceAll(/\(/g, 'c')
-        .replaceAll(/ü/g, 'u')
-        .replaceAll(/[^a-z ]/g, '');
```

Comment:
Do we want to remove this. Without regex this won't cause false positives.

Reply:
Yes, point of this is to just keep the thing simplistic